substratevm/src/com.oracle.objectfile/src/com/oracle/objectfile/pecoff/cv/CVSymbolSectionImpl.java

```diff
@@ -186,4 +186,42 @@ int getCurrentOffset() {
     void addRecord(CVSymbolRecord record) {
         cvRecords.add(record);
     }
+
+    /**
+     * Mark an offset:segment relocation site for linker or loader fixup.
+     *
+     * @param buffer output buffer
+     * @param initialPos position of fixup in output buffer
+     * @param symbolName symbolname to reference
+     * @return new position in output buffer
+     */
+    public int markRelocationSite(byte[] buffer, int initialPos, String symbolName) {
+        int pos = initialPos;
+        pos = markRelocationSite(buffer, pos, symbolName, RelocationKind.SECREL_4, 0);
+        pos = markRelocationSite(buffer, pos, symbolName, RelocationKind.SECTION_2, 0);
+        return pos;
```

Comment:
Could be simplified a bit.
```suggestion
        return markRelocationSite(buffer, pos, symbolName, 0);
```

Reply:
Done.